pandas-stubs/core/reshape/concat.pyi

```diff
@@ -12,15 +15,18 @@ from pandas import (
 
 from pandas._typing import HashableT
 
+_HashableT2 = TypeVar("_HashableT2", bound=Hashable)
+_HashableT3 = TypeVar("_HashableT3", bound=Hashable)
```

Comment:
Now that I merged in #377 you should use those

Reply:
Fixed everywhere.

Reply:
Fixed.